prime-router/src/main/kotlin/cli/helpers/HL7DiffHelper.kt

```diff
@@ -97,6 +97,7 @@ class HL7DiffHelper {
                     }
                     inputFields.foldIndexed(differences) { index, differenceAccumulator, input ->
                         try {
+                            // TODO: Check how many fields there are (if there is just one no need to include the secondary field
```

Comment:
I think we should be good to remove this comment?

Reply:
😅 Added this so I didn't forget and forgot to remove